sklearn/metrics/scorer.py

```diff
@@ -66,6 +66,11 @@ def __repr__(self):
                    "" if self._sign > 0 else ", greater_is_better=False",
                    self._factory_args(), kwargs_string))
 
+    def _set_score_func_parameters(self, **kwargs):
```

Comment:
added a new function in base scorer to allow modifying score_func parameters. originally, it was designed such that score_func parameters can be set only at the time of initializing the make_scorer. but get_scorer has them pre-defiend so a new function was needed.